### PR TITLE
{bio}[foss/2019a] CONCOCT v1.1.0 w/ Python 2.7.15 + 3.7.2

### PR DESCRIPTION
**Pull request by Darkless012**
(created using `eb --new-pr`)
requires ~~#9153~~ (Biopython), ~~#9164~~ (MEGAHIT), ~~#9159~~ (picard), ~~#9167~~ (CheckM)

### CODE REVIEW

**Comment by Darkless012** [comment]
Test report by @Darkless012
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3211.victini.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/154f167654bce3834d1f09c8e85950d0 for a full test report.

**Comment by boegel** [comment]
(close/re-open to re-trigger tests)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/21b2328ea62df4b44a0d8d641b49d86f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3133.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/ce8628ed7e9f1a3e161cbd0636595f32 for a full test report.

File: easybuild/easyconfigs/c/CONCOCT/CONCOCT-1.1.0-foss-2019a-Python-2.7.15.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+source_urls = ['https://github.com/BinPro/CONCOCT/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['00aecacb4b720ac123a63e65072c61e0b5a8690d844c869aaee4dbf287c82888']
+
+dependencies = [
+    ('Python', '2.7.15'),
+    ('GSL', '2.5'),
+    # gslcblas
+    ('Biopython', '1.74'),
```
Comment (boegel) [inline easybuild/easyconfigs/c/CONCOCT/CONCOCT-1.1.0-foss-2019a-Python-2.7.15.eb]: @Darkless012 You'll need to stick to Biopython 1.73 to make the tests pass...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/e06859b33c8152dff3f6dcfd5b3ad4aa for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3149.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/9fb2ca24ad9600c816c5595173c61ea1 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (1 easyconfigs in this PR)
node2413.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/a27f21179e1e993e229da5943ad2d13e for a full test report.

**Comment by boegel** [comment]
Going in, thanks @Darkless012!

**Pull request merged**